-------------------------------------------------------------------------------
Test set: com.otto.dao.UsuarioDAOTest
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.001 s <<< FAILURE! - in com.otto.dao.UsuarioDAOTest
testInsertarYConsultarUsuario  Time elapsed: 0 s  <<< ERROR!
java.lang.NullPointerException: Cannot invoke "java.sql.Connection.prepareStatement(String)" because "conn" is null
	at com.otto.dao.UsuarioDAOTest.testInsertarYConsultarUsuario(UsuarioDAOTest.java:17)

